iceoryx_examples/iceperf/iceperf_leader.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2019 - 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2020 - 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "iceperf_leader.hpp"
+#include "iceoryx.hpp"
+#include "iceoryx_c.hpp"
+#include "iceoryx_posh/popo/publisher.hpp"
+#include "iceoryx_posh/popo/subscriber.hpp"
+#include "iceoryx_posh/runtime/posh_runtime.hpp"
+#include "iceoryx_utils/cxx/convert.hpp"
+#include "mq.hpp"
+#include "topic_data.hpp"
+#include "uds.hpp"
+
+#include <iomanip>
+#include <iostream>
+
+//! [use constants instead of magic values]
+constexpr const char APP_NAME[]{"iceperf-bench-leader"};
+constexpr const char PUBLISHER[]{"Hardy"};
```

Comment:
Wouldn't it be more consistent to call them now also `leader` and `follower`? Let's say I take a look at the benchmark in the introspection (or system files in `/dev/mq`) then I just see `Laurel` and `Hardy` and wonder where my `leader` and `follower` is.

Reply:
You are right, but ... but ... 😢